napari/_qt/qt_main_window.py

```diff
@@ -119,6 +123,7 @@ def __init__(self, viewer, *, show: bool = True):
         self._qt_center.layout().addWidget(self.qt_viewer)
         self._qt_center.layout().setContentsMargins(4, 0, 4, 0)
 
+        self.qt_viewer.viewer.theme = SETTINGS.application.theme
```

Comment:
Just a thought here, once we have settings we might just drop `viewer.theme` entirely. I don't see why we'd need to remember both `viewer.theme` and `SETTINGS.application.theme`. We can wait to deprecate this gracefully etc, but it's just a thought.

Reply:
One additional thought then is what does it mean for a plugin to have access to napari settings then to read - this has been important for the styling of the `napari-animation` and `napari-console` plugins (we don't have to solve or discuss this now, but just noting it in passing)

Reply:
>the only reason is if we'd want to support different themes on two viewer instances. I don't think we should. it dramatically simplifies things if there is a global theme that anything can query at any given time

Agreed we would not want to support this. When several VSCode instances are open, changing this setting will update both